mulog-core/src/com/brunobonacci/mulog/jvm_metrics.clj

```diff
@@ -0,0 +1,66 @@
+(ns ^{:author "Pablo Reszczynski (@Pablo Reszczynski)"
+      :doc "Module for sampling some JVM metrics"}
+  com.brunobonacci.mulog.jvm-metrics
+  (:require [clojure.spec.alpha :as s])
+  (:import (java.lang.management MemoryMXBean
+                                 MemoryPoolMXBean)))
+
+(defn jvm-sample [opts] nil)
+
+(s/def :memory/total.init int?)
+(s/def :memory/total.used int?)
+(s/def :memory/total.max int?)
+(s/def :memory/total.committed int?)
+(s/def :memory/heap.init int?)
+
+(s/def ::captured-memory
+  (s/keys :req [:memory/total.init
+                :memory/total.used
+                :memory/total.max
+                :memory/total.commited
+                :memory/heap.init]))
+
+(defn- get-usage-ratio [usage]
+  (/ (.getUsed usage) (.getMax usage)))
```

Comment:
Please hint the type to avoid reflection.
use `lein check` to verify that no reflection is used

Reply:
done